src/garvis/local_language_runtime.py

```
@@ -0,0 +1,216 @@
+    def __init__(self, config: LocalRuntimeConfig) -> None:
+        config.validate()
+        self.config = config
+
+    def respond(self, message: str) -> str:
+        prompt = render_local_prompt(classify_request(message))
+        command = [
+            str(self.config.engine), "-m", str(self.config.model),
+            "-c", str(self.config.context_size), "-ngl", str(self.config.gpu_layers),
+        ]
+        try:
+            completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)